rpm/SPECS/openresty-pcre.spec

```diff
@@ -0,0 +1,119 @@
+Name:       openresty-pcre
+Version:    8.39
+Release:    1%{?dist}
+Summary:    Perl-compatible regular expression library for OpenResty
+
+Group:      System Environment/Libraries
+
+License:    BSD
+URL:        http://www.pcre.org/
+Source0:    ftp://ftp.csx.cam.ac.uk/pub/software/programming/pcre/pcre-%{version}.tar.bz2
+
+# http://vcs.pcre.org/pcre/code/trunk/pcregrep.c?view=patch&r1=1658&r2=1657&pathrev=1658
+Patch0:     patch_1658.diff
+
+BuildRequires:  readline-devel
+BuildRequires:  autoconf
+BuildRequires:  automake
```

Comment:
I don't think we need the autotools since we can invoke the `./configure` script directly.